docs/_posts/examples/3400-01-31-data-driven-lines.html

```diff
@@ -0,0 +1,266 @@
+---
+layout: example
+category: example
+title: Style lines using data-driven styling and the identity property function.
```

Comment:
All that's visible of this title in the page's fixed-width sidebar is 

> Style lines using data-driven styling an..

Is there a way we can word the title to capture the intent of the example ("Add identity function example") 
visible in the sidebar? Perhaps one of these titles:

 > Style lines using an identity property function
 > Style lines using an identity function
 > Style using an identity function
 > Style using an identity property function